tests/dat/actions/openFiles.js

```diff
@@ -22,7 +22,7 @@ function main(params) {
     console.log("opened files = ", openFiles.length);
 
     openFiles.forEach(function(fh) {
-        fs.close(fh);
+        fs.close(fh, (err) => {} );
```

Comment:
Should this be `closeSync` instead? If not: Can we safely omit the callback function?

Reply:
The empty callback was added since this parameter was optional for nodejs:6 but is now required for nodejs:10. Using an empty callback funtion sounded to be the default behaviour when this parameter is not present in nodejs:6. Explicitly defining the empty callback now, works with both nodejs:6 and nodejs:10.
You are right, making this a `closeSync ` call will most likely not change the test behaviour, since this is cleanup code at the end of the test. But I would need to try it first with both node versions.